File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.js
function solution(phone_number) {
    var answer = '';
    
    const length = phone_number.length
    
    for(let i=0; i<length; i++){
        
        if(i<length-4){
            answer +=  '*';
            
        } else{
            answer +=  phone_number[i];
        }
    }
    
    console.log(answer);
    
    return answer;
}

File: 프로그래머스/1/12948. 핸드폰 번호 가리기/핸드폰 번호 가리기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string phone_number) {
    string answer = phone_number;
    
    for(int i=0; i<phone_number.length()-4; i++){
        
        phone_number[i] = '*';
        
    }
    answer = phone_number;
    
    return answer;
}